tests/integration-tests/tests/createami/test_createami.py

```diff
@@ -55,29 +58,13 @@ def test_createami(region, os, instance, request, pcluster_config_reader, vpc_st
         + networking_args
     )
 
-    assert_that(
-        any(
-            "downloading https://{0}-aws-parallelcluster.s3".format(region).lower() in s.lower()
-            for s in pcluster_createami_result.stdout.split("\n")
-        )
-    ).is_true()
-    assert_that(
-        any("chef.io/chef/install.sh".lower() in s.lower() for s in pcluster_createami_result.stdout.split("\n"))
-    ).is_false()
-    assert_that(
-        any("packages.chef.io".lower() in s.lower() for s in pcluster_createami_result.stdout.split("\n"))
-    ).is_false()
-    assert_that(
-        any(
-            "Thank you for installing Cinc Client".lower() in s.lower()
-            for s in pcluster_createami_result.stdout.split("\n")
-        )
-    ).is_true()
-    assert_that(
-        any("Starting Cinc Client".lower() in s.lower() for s in pcluster_createami_result.stdout.split("\n"))
-    ).is_true()
-
-    assert_that(pcluster_createami_result.stdout).does_not_contain("No custom AMI created")
+    stdout_lower = pcluster_createami_result.stdout.lower()
+    assert_that(stdout_lower).contains("downloading https://{0}-aws-parallelcluster.s3".format(region).lower())
+    assert_that(stdout_lower).does_not_contain("chef.io/chef/install.sh".lower())
+    assert_that(stdout_lower).does_not_contain("packages.chef.io".lower())
```

Comment:
I am not sure do we need to lower the string that doesn't contain upper case?

Reply:
Done